src/routes/chat/Chat.js

```diff
@@ -1,37 +1,84 @@
-import React, { memo, useState, useCallback } from 'react';
+import React, { memo, useState, useCallback, useEffect } from 'react';
 import get from 'lodash/get';
-import PropTypes from 'prop-types';
+import property from 'lodash/property';
+// import PropTypes from 'prop-types';
```

Comment:
Why don't you define propTypes for `Chat` component?

Reply:
When I applied it to messages Eslint show me error "That propTypes defined but never used". The array checked in Messages component. For what element 
useful to applied prop types?